chapter11/dtj/ex2

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/perl
+
+use DateTime;
+use Scalar::Util qw(looks_like_number); # Needed declaring, unlike for some others
```

Comment:
Some methods are exported by default, while some have to be specified. The details come up in the next book.